.gitignore

```diff
@@ -9,6 +9,9 @@ Icon
 # Thumbnails
```

Comment:
Move this file to front-end folder. Remember, all the files live in the front-end folder